testing/test_cases.go

```diff
@@ -109,25 +109,18 @@ func TcDevice(tcs []TestCase) (*Device, error) {
 	quoteResponses := map[[labi.TdReportSize]byte]any{}
 	for _, tc := range tcs {
 		reportResponses[tc.Input] = &GetReportResponse{
-			Resp: labi.TdxReportABI{
-				Data: tc.Report,
+			Resp: labi.TdxReportReq{
+				TdReport: tc.Report,
 			},
 			EsResult: tc.EsResult,
 		}
-		var idQuote [labi.TdIDQuoteSize]byte
```

Comment:
I think after this change, TdIDQuoteSize will be unused and can be removed.
Also TdQuoteBufSize will be unused after TdIDQuoteSize is removed.

Reply:
Yes, it is already removed from linux_abi.go
Did I miss anything?